lib/matplotlib/text.py

```diff
@@ -135,6 +136,7 @@ def __init__(self,
         super().__init__()
         self._x, self._y = x, y
         self._text = ''
+        self._antialiased = None
```

Comment:
Can we start with the rcparam value here (so it is always type bool) and then it will get set to the user value (if any) in the next call.

Reply:
Done